content/en/agent/_index.md

```diff
@@ -29,279 +29,25 @@ aliases:
 Agent v6 is available. <a href="/agent/faq/upgrade-to-agent-v6">Upgrade to the newest version</a> to benefit from all new functionality.
```

Comment:
Since this page is getting updates, let's change line 17 to:
`Why install the Datadog Agent on cloud instances?`